### PR TITLE
Split Supports.{cpp,h} into two and refactor the wooden supports

### PR DESCRIPTION
**Pull request by Gymnasiast**
Most of this is code being moved around, except for WoodenSupports.cpp, where additional refactoring has taken places.

This has been mostly done to make the mapping between wooden support types/subtypes and their images clearer. It also gets rid of an `if` when accessing `SupportsDescriptors` (previously known as `Byte97B23C`), hopefully making for code that is a hair faster.

### CODE REVIEW

**Review by ZehMatt — APPROVED** [review]
Looks good, should probably get a bit of testing just to make sure.

**Comment by ZeeMaji** [comment]
I went ahead and looked at about 10 different parks with varying types of wooden rides on this branch and they all seem to be rendering as they should, i didn't see any anomalies.

**Review by ZeeMaji — APPROVED** [review]

File: src/openrct2/paint/support/Generic.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*****************************************************************************
```
Comment (tupaschoal) [inline src/openrct2/paint/support/Generic.h:1]: `Supports.h` seemed like a better name, I guess

Comment (Gymnasiast) [reply src/openrct2/paint/support/Generic.h:1]: I want to clean that up later anyway - I think it is just coincidentally the same value. Just didn’t want to change too much in one go.

**Pull request merged**